docs/source/status.rst

```diff
@@ -111,14 +111,24 @@ Data Types
 | Run-End Encoded   | ✓     |       | ✓     |            |       |       |       |       |
 +-------------------+-------+-------+-------+------------+-------+-------+-------+-------+
 
++---------------------+-------+-------+-------+------------+-------+-------+-------+-------+
+| Canonical           | C++   | Java  | Go    | JavaScript | C#    | Rust  | Julia | Swift |
+| Extension types     |       |       |       |            |       |       |       |       |
++=====================+=======+=======+=======+============+=======+=======+=======+=======+
+| FixedShapeTensor    | ✓     |       |       |            |       |       |       |       |
++---------------------+-------+-------+-------+------------+-------+-------+-------+-------+
+| VariableShapeTensor |       |       |       |            |       |       |       |       |
```

Comment:
These are the C++ and Python internal names, should we use the prose spelling instead? "Fixed shape tensor" and "Variable shape tensor"

Reply:
Good point, thanks for catching this! Fixed.